core/src/modules/configWriter.ts

```diff
@@ -108,6 +108,7 @@ export namespace ConfigWriter {
       properties: await Property.findAll(queryParams),
       groups: await Group.findAll(queryParams),
       destinations: await Destination.findAll(queryParams),
+      records: await GrouparooRecord.findAll({ include: [GrouparooModel] }),
```

Comment:
Where were the `null` objects coming from? Did `getConfigObject()` for an invalid record return it? 

I'm guessing this fixes the `null` or `undefined` objects because the loop that goes over these `queries` and calls `getConfigObject()` does some validations?

Reply:
Yep! The checks below filter away the null objects.  Before this PR, the records writer wasn't using those checks - so you could get null sample records - eg, with `user_id=-1` (when the primary key goes null)